wdio.conf.js

```diff
@@ -20,7 +20,7 @@ exports.config = {
     // directory is where your package.json resides, so `wdio` will be called from there.
     //
     specs: [
-        './specs/**/*.js'
+        './specs/**/claims/*.js'
```

Comment:
this line not commit, this code is only for your test change to ` './specs/**/*.js' when you add a commit 
`